docs/content/howto/helm_chart_redhat_certification.md

```diff
@@ -69,7 +77,7 @@ The steps for certification are as follows:
 
 ```yaml
 # Build the manager binary
-FROM quay.io/operator-framework/helm-operator:v1.3.2
+FROM quay.io/operator-framework/helm-operator:v1.7.1
```

Comment:
Suggestion - here and other similar places, may be good idea to add in comment that use version of your choice or latest version as possible - something like "# Build the manager binary. Use the latest version or version of your choice".

Reply:
Good point. Updated.